scopes/design/ui/tooltip/tippy.module.scss

```diff
@@ -12,11 +12,19 @@ $textColor: var(--bit-text-color, #fff);
 
     .tippy-content {
       padding: 0.85em;
-      white-space: nowrap;
+      &[data-breakline='false'] {
+        white-space: nowrap;
+      }
```

Comment:
I think it's better to leave `white-space: nowrap` as default. I'm not sure the dome will always include [`data-breakline='false']